command_test.go

```diff
@@ -2161,3 +2161,90 @@ func TestSetContextPersistentPreRun(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+const VERSION_FLAG = "--version"
+const HELP_FLAG = "--help"
```

Comment:
I just noticed the linter is failing on these two names, it says: "don't use ALL_CAPS in Go names; use CamelCase"

Reply:
fixed